service/Microsoft.DSX.ProjectTemplate.Test/Tests/Integration/Library/GetLibraryTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FluentAssertions;
+using Microsoft.DSX.ProjectTemplate.Data.DTOs;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.DSX.ProjectTemplate.Test.Tests.Integration.Library
+{
+    [TestClass]
+    [TestCategory("Libraries - Get")]
+    public class GetLibraryTests : BaseLibraryTest
+    {
+        [TestMethod]
+        public async Task GetAllLibrary_Valid_Success()
```

Comment:
this test failed, I tracked the error and it seems that the function can't read the libraries from the database .